docs/topics/api/v4_frozen/index.rst

```diff
@@ -0,0 +1,51 @@
+.. _v4-api-index:
+
+=================
+External API (V4)
+=================
+
+This shows you how to use the `addons.mozilla.org <https://addons.mozilla.org/en-US/firefox/>`_
```

Comment:
Would it be worth adding something here (as the main page for the API) that explains that this is the current frozen version of the API?

Reply:
I added a sentence and link back to the main (v5) api page where it explains the versions
https://github.com/mozilla/addons-server/pull/16303/commits/97c1654fb9e739eca3fd3ff3d297eaa054090a30